### DISCUSSION TITLE
Problems detecting number transitions

### DISCUSSION THREAD

**Discussion by JWRu** [jomjol/AI-on-the-edge-device#2669] [Q&A]
I have problems during number transitions. The reading 679 (see photo attached) is recognized as 669 or 659.
I have used all digital processing models - no improvement.
Post processing previous value is enabled - analog/digital transition start is set to 9.2
Any recommendations?
![Water Meter 01](https://github.com/jomjol/AI-on-the-edge-device/assets/89386880/83d5714a-5489-44a7-b9d0-eae3a0a26187)

**Comment by schneich** [comment]
Hi @JWRu ,

I had a similar problem. Turned out, my misreadings were due to reflections on the numbers - similar to your picture.
https://github.com/jomjol/AI-on-the-edge-device/discussions/1949#discussioncomment-7355535

I changed the light distribution and this seems to have solved the issue.

Chris
